### ISSUE TITLE
"generate price" in stock

### ISSUE THREAD

**Issue by Thorgal-The-Viking** [nicho92/MtgDesktopCompanion#137]
![image](https://user-images.githubusercontent.com/70637347/93639528-6ed75480-f9f9-11ea-8f7b-c73a0bdadd93.png)

Can I have generate price in stock look at another pricer instead of TCGPlayer?
I would like it to generate a price based on my Magic Cardmarket settings. ( considering country, quality of card ... )

**Comment by nicho92** [comment]
Price generator in stock module use "Dashboard" provider. 
Effectively, it could be a good idea to use price provider.

**Comment by Thorgal-The-Viking** [comment]
Yes it would be really nice as European prices are totally different
